Sources/StreamChat/Query/Sorting/ChannelListSortingKey.swift

```diff
@@ -5,55 +5,53 @@
 import Foundation
 
 /// `ChannelListSortingKey` is keys by which you can get sorted channels after query.
-public enum ChannelListSortingKey: String, SortingKey {
+public struct ChannelListSortingKey: SortingKey, Equatable {
     /// The default sorting is by the last massage date or a channel created date. The same as by `updatedDate`.
-    case `default` = "defaultSortingAt"
+    public static let `default` = Self(keyPath: \.defaultSortingAt, localKey: "defaultSortingAt", remoteKey: "updated_at", canUseAsDBSortDescriptor: true)
     /// Sort channels by date they were created.
-    case createdAt
+    public static let createdAt = Self(keyPath: \.createdAt, remoteKey: "created_at", canUseAsDBSortDescriptor: true)
     /// Sort channels by date they were updated.
-    case updatedAt
+    public static let updatedAt = Self(keyPath: \.updatedAt, remoteKey: "updated_at", canUseAsDBSortDescriptor: true)
     /// Sort channels by the last message date..
-    case lastMessageAt
+    public static let lastMessageAt = Self(keyPath: \.lastMessageAt, remoteKey: "last_message_at", canUseAsDBSortDescriptor: true)
     /// Sort channels by number of members.
-    case memberCount
+    public static let memberCount = Self(keyPath: \.memberCount, remoteKey: "member_count", canUseAsDBSortDescriptor: true)
```

Comment:
Can the Remove Key actually be the ChannelPayload Coding Keys? For better compile safe keys?